src/openfermion/ops/_qubit_operator.py

```diff
@@ -65,167 +60,11 @@ class QubitOperator(object):
     .. code-block:: python
 
         hamiltonian = 0.5 * QubitOperator('X0 X5') + 0.3 * QubitOperator('Z0')
-
-    Attributes:
-        terms (dict): **key**: A term represented by a tuple containing all
-                      non-trivial local Pauli operators ('X', 'Y', or 'Z').
-                      A non-trivial local Pauli operator is specified by a
-                      tuple with the first element being an integer
-                      indicating the qubit on which a non-trivial local
-                      operator acts and the second element being a string,
-                      either 'X', 'Y', or 'Z', indicating which non-trivial
-                      Pauli operator acts on that qubit. Examples:
-                      ((1, 'X'),) or ((1, 'X'), (4,'Z')) or the identity ().
-                      The tuples representing the non-trivial local terms
-                      are sorted according to the qubit number they act on,
-                      starting from 0.
-                      **value**: Coefficient of this term as a (complex) float
     """
-
-    def __init__(self, term=None, coefficient=1.):
-        """
-        Inits a QubitOperator.
-
-        The init function only allows to initialize one term. Additional terms
-        have to be added using += (which is fast) or using + of two
-        QubitOperator objects:
-
-        Example:
-            .. code-block:: python
-
-                ham = ((QubitOperator('X0 Y3', 0.5)
-                        + 0.6 * QubitOperator('X0 Y3')))
-                # Equivalently
-                ham2 = QubitOperator('X0 Y3', 0.5)
-                ham2 += 0.6 * QubitOperator('X0 Y3')
-
-        Note:
-            Adding terms to QubitOperator is faster using += (as this is done
-            by in-place addition). Specifying the coefficient in the __init__
-            is faster than by multiplying a QubitOperator with a scalar as
-            calls an out-of-place multiplication.
-
-        Args:
```

Comment:
Shouldn't we keep these docs?

Reply:
Yes, in hindsight I went overboard with trimming the docs. I'll add some back in.